tests/store/test_models.py

```diff
@@ -52,6 +52,7 @@ def test_charm(self):
         self.assertEqual(charm.homepage, "https://launchpad.net/apache2-charm")
         self.assertTrue(charm.icon.endswith("apache2-26/icon.svg"))
         self.assertEqual(charm.id, "cs:apache2-26")
+        self.assertEqual(charm.canonical_url, "https://jaas.ai/apache2")
```

Comment:
Can you also add a test for the non promulgated charms?

Reply:
Not worth the new suite for a codebase that is about to be replaced.